_config.yml

```diff
@@ -1,9 +1,9 @@
 # Site wide configuration
 
-title: "reflectoring"
+title: "reflectosadssadring"
```

Comment:
Also, please update links to favicons in default.html to this:

```
<link rel="apple-touch-icon" sizes="180x180" href="/images/logo/apple-icon.png">
<link rel="icon" type="image/png" href="/images/logo/favicon-32x32.png" sizes="32x32">
<link rel="icon" type="image/png" href="/images/logo/favicon-16x16.png" sizes="16x16">
```
Currently, the favicons cannot be found, which is why the CI-Job failed.